homeassistant/components/bosch_alarm/alarm_control_panel.py

```diff
@@ -36,6 +36,17 @@ async def async_setup_entry(
 
 PARALLEL_UPDATES = 0
 
+BG_PANELS = {
```

Comment:
This kind of hardcoding does not seem like a good idea.
Much more robust to have the library report whether the feature of interest is supported directly.